### PR TITLE
tests: Increase timeout to crictl calls on kata monitor tests

### PR DESCRIPTION
**Pull request by GabyCT**
This PR increases the timeout to crictl calls on kata monitor tests to avoid to hit issues every now and avoid random failures. This PR is very similar to PR #7640.

### CODE REVIEW

**Review by danmihai1 — APPROVED** [review]

File: tests/functional/kata-monitor/kata-monitor-tests.sh [hunk 1]
```diff
@@ -144,9 +144,9 @@ start_workload() {
 	sbfile="$(create_sandbox_json)"
 	cntfile="$(create_container_json)"
 
-	POD_ID=$(sudo crictl runp $args $sbfile)
-	CID=$(sudo crictl create $POD_ID $cntfile $sbfile)
-	qcrictl start $CID
+	POD_ID=$(sudo crictl --timeout=20s runp $args $sbfile)
```
Comment (dborquez) [inline tests/functional/kata-monitor/kata-monitor-tests.sh:147]:
```suggestion
	POD_ID=$(sudo crictl --timeout=${TO} runp $args $sbfile)
```

File: tests/functional/kata-monitor/kata-monitor-tests.sh [hunk 2]
```diff
@@ -144,9 +144,9 @@ start_workload() {
 	sbfile="$(create_sandbox_json)"
 	cntfile="$(create_container_json)"
```
Comment (dborquez) [inline tests/functional/kata-monitor/kata-monitor-tests.sh:146]: readonly TO="20s"

**Review by dborquez — CHANGES_REQUESTED** [review]
Because the assigned timeout is the same in all cases, could you set a timeout using a variable?

**Review by dborquez — APPROVED** [review]
LGTM, thanks @GabyCT

**Pull request merged**